mcs/class/Mono.Compiler/Mono.Compiler/RuntimeInformation.cs

```diff
@@ -5,10 +5,10 @@
 namespace Mono.Compiler {
 	public class RuntimeInformation : IRuntimeInformation {
 		[MethodImplAttribute(MethodImplOptions.InternalCall)]
-		extern static InstalledRuntimeCode mono_install_compilation_result (int compilationResult, NativeCodeHandle codeHandle);
+		extern static InstalledRuntimeCode mono_install_compilation_result (int compilationResult, MethodInfo methodInfo, NativeCodeHandle codeHandle);
 
-		public InstalledRuntimeCode InstallCompilationResult (CompilationResult compilationResult, NativeCodeHandle codeHandle) {
-			return mono_install_compilation_result ((int) compilationResult, codeHandle);
+		public InstalledRuntimeCode InstallCompilationResult (CompilationResult compilationResult, MethodInfo methodInfo, NativeCodeHandle codeHandle) {
+			return mono_install_compilation_result ((int) compilationResult, methodInfo, codeHandle);
```

Comment:
can't we get the runtime method handle here in managed and avoid a runtime invoke down in `mono_install_compilation_result` ?